vault/expiration.go

```diff
@@ -1806,6 +1822,67 @@ func (m *ExpirationManager) deleteLockForLease(id string) {
 	m.lockPerLease.Delete(id)
 }
 
+func (m *ExpirationManager) sendEventByID(eventType logical.EventType, leaseID string) {
+	le, err := m.loadEntry(context.Background(), leaseID)
+	if err != nil {
+		m.logger.Warn("Error fetching lease information", "error", err)
+		return
+	}
+	m.sendEvent(eventType, le)
+}
+
+func (m *ExpirationManager) sendEvent(eventType logical.EventType, le *leaseEntry) {
+	data := &logical.EventData{}
+	id, err := uuid.GenerateUUID()
+	if err != nil {
+		m.logger.Warn("Error creating uuid", "error", err)
+		return
+	}
+	ns := namespace.RootNamespace
+	if le.namespace != nil {
+		ns = le.namespace
+	}
+	data.Id = id
+	data.EntityIds = []string{le.ClientToken}
```

Comment:
Is `ClientToken` a secret (it very much sounds like it)? If so, we shouldn't put it in the event

Reply:
I thought it wasn't, for some reason, but upon further inspection, it appears to possibly be a secret. I'll remove it.